src/jit/lclvars.cpp

```diff
@@ -133,14 +133,13 @@ void                Compiler::lvaInitTypeRef()
     // Change the compRetNativeType if we are returning a struct by value in a register
     if (!hasRetBuffArg && varTypeIsStruct(info.compRetNativeType))
     {
-#ifdef _TARGET_ARM_
-        // TODO-ARM64-NYI: HFA
+#if FEATURE_MULTIREG_RET && defined(FEATURE_HFA)
```

Comment:
@briansull  offline explained that right now this logic (meant for hfa return in multiple regs) is not enabled for ARM64.  With this check-in to maintain similar condition FEATURE_MULTIREG_RET  condition got added since it is defined as 1 only for arm32 and zero for arm64.  @briansull subsequently would enable this when FEATURE_MULTIREG_RET is set to 1.  At that point we should clean this #ifdef and all such occurrences.